kyo-prelude/shared/src/test/scala/kyo/ChoiceTest.scala

```diff
@@ -81,7 +83,7 @@ class ChoiceTest extends Test:
         end try
     }
 
-    "large number of suspensions" in pendingUntilFixed {
+    "large number of suspensions" taggedAs notNative in pendingUntilFixed {
```

Comment:
Scala Native crashes and reports a failures due to the stack overflow even with `pendingUntilFixed`.